src/gridfs-stream/download.ts

```diff
@@ -372,9 +428,15 @@ function handleStartOption(stream: any, doc: any, options: any) {
 
     return options.start - stream.s.bytesRead;
   }
+  throw new Error('no option available');
```

Comment:
I think we should use MongoError here and in general we should try to use only MongoError, we want to eventually have that be our only error type or subclasses of it. Also [nit] maybe `'No start option defined'` slightly more clear message.